hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py

```diff
@@ -447,12 +447,18 @@ async def _process_user_stream_event(self, event_message: Dict[str, Any]):
             # update position
             for asset in update_data.get("P", []):
                 trading_pair = asset["s"]
+                try:
```

Comment:
This is not needed since you should have the full map in the connector. Is not related to initialization